test/functions/diff.js

```diff
@@ -19,10 +19,19 @@ test('diff versions test', (t) => {
     ['1.1.0', '1.1.0-pre', 'minor'],
     ['1.1.0-pre-1', '1.1.0-pre-2', 'prerelease'],
     ['1.0.0', '1.0.0', null],
+    ['1.0.0-1', '1.0.0-1', null],
     ['0.0.2-1', '0.0.2', 'patch'],
+    ['0.0.2-1', '0.0.3', 'patch'],
+    ['0.0.2-1', '0.1.0', 'minor'],
+    ['0.0.2-1', '1.0.0', 'major'],
     ['0.1.0-1', '0.1.0', 'minor'],
     ['1.0.0-1', '1.0.0', 'major'],
-    ['0.0.0-1', '0.0.0', 'prerelease'],
```

Comment:
Is this the only change in behavior that's currently covered in tests?

Reply:
The new ones I added in the first commit had different values before the fix as they were classed as pre* versions before